packages/client/src/components/workspace/flexlayout.css

```diff
@@ -7,56 +7,66 @@
     --font-size: 0.875rem;
     --smss-tab-border-radius: 4px;
 }
-/* 
-  Sidebar tabs
- */
 .flexlayout__border_left {
-    border-right: none;
-    background: none;
-    padding-bottom: 40px;
+    margin-right: 10px;
+    border-radius: 12px !important;
```

Comment:
Important is a pretty bad css property to use, are you sure you need to use it as often as you are here? I want to make sure that we are not shooting ourselves in the foot.